messenger.rst

```diff
@@ -623,103 +626,28 @@ you can disable them like this:
-                arguments: ['default']
-
-    .. code-block:: xml
-
-        <!-- config/services.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd">
-
-            <services>
-                <!-- Step 1: a factory class is registered as a service with the required
-                     dependencies to instantiate a middleware -->
-                <service id="doctrine.orm.messenger.middleware_factory.transaction"
-                    class="Symfony\Bridge\Doctrine\Messenger\DoctrineTransactionMiddlewareFactory">
-
-                    <argument type="service" id="doctrine" />
-                </service>
-
-                <!-- Step 2: an abstract definition that will call the factory with default
-                     arguments or the ones provided in the middleware config -->
-                <service id="messenger.middleware.doctrine_transaction_middleware"
-                    class="Symfony\Bridge\Doctrine\Messenger\DoctrineTransactionMiddleware"
-                    abstract="true">
-
-                    <factory service="doctrine.orm.messenger.middleware_factory.transaction"
-                        method="createMiddleware" />
-                    <argument>default</argument>
-                </service>
-            </services>
-        </container>
-
-    .. code-block:: php
-
-        // config/services.php
-        use Symfony\Bridge\Doctrine\Messenger\DoctrineTransactionMiddleware;
-        use Symfony\Bridge\Doctrine\Messenger\DoctrineTransactionMiddlewareFactory;
-        use Symfony\Component\DependencyInjection\Reference;
-
-        // Step 1: a factory class is registered as a service with the required
-        // dependencies to instantiate a middleware
-        $container
-            ->register('doctrine.orm.messenger.middleware_factory.transaction', DoctrineTransactionMiddlewareFactory::class)
-            ->setArguments(array(new Reference('doctrine')));
-
-        // Step 2: an abstract definition that will call the factory with default
-        // arguments or the ones provided in the middleware config
-        $container->register('messenger.middleware.doctrine_transaction_middleware', DoctrineTransactionMiddleware::class)
-            ->setFactory(array(
-                new Reference('doctrine.orm.messenger.middleware_factory.transaction'),
-                'createMiddleware'
-            ))
-            ->setAbstract(true)
-            ->setArguments(array('default'));
-
-The "default" value in this example is the name of the entity manager to use,
-which is the argument expected by the
-``Symfony\Bridge\Doctrine\Messenger\DoctrineTransactionMiddlewareFactory::createMiddleware`` method.
+factories.
```

Comment:
there's no factory anymore for Doctrine, so not sure the example below applies, does it?